src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -327,7 +327,7 @@ def load_arguments(self, _):
         VirtualMachineEvictionPolicyTypes = self.get_models('VirtualMachineEvictionPolicyTypes', resource_type=ResourceType.MGMT_COMPUTE)
         c.argument('name', name_arg_type)
         c.argument('nat_backend_port', default=None, help='Backend port to open with NAT rules.  Defaults to 22 on Linux and 3389 on Windows.')
-        c.argument('single_placement_group', arg_type=get_three_state_flag(), help="Enable replicate using fault domains within the same cluster. Default to 'false' for any zonals, or with 100+ instances"
+        c.argument('single_placement_group', arg_type=get_three_state_flag(), help="Limit the scale set to a single placement group with a max size of 100 instances."
```

Comment:
Is the 100 instance limit still applicable?

Reply:
I don't know the answer, but per conversation with CRP, we shall remove all the client side validations on this, and leverage the server end error. So @adewaleo, please try out and see whether or not this limitation is no longer valid, or the server error is clear now

Reply:
I will try out. I was just wary of creating a VMSS with a lot of instances. 

Forgot to update the help text. Will test and also update the help text.

Reply:
cc: @yugangw-msft